### PR TITLE
Add a profile preference for inverting thumbnails app-wide

### LINKED ISSUE #849: Toggle negative / positive image display sitewide by User
Allow, on the Profile page, the user to set the display of images as negative (sky background white, source flux black) or positive (sky background black, source flux white).

### PR DESCRIPTION
**Pull request by kmshin1397**
Closes #849 
invert_images

### CODE REVIEW

File: static/js/components/RecentSources.jsx [hunk 1]
```diff
@@ -24,16 +24,18 @@ export const useSourceListStyles = makeStyles((theme) => ({
   stampContainer: {
     display: "contents",
   },
-  stamp: {
+  stamp: (props) => ({
```
Comment (stefanv) [inline static/js/components/RecentSources.jsx:27]: I'm surprised the linter did not make you unpack that into `({invertThumbnails})`

**Review by stefanv — APPROVED** [review]

**Review by profjsb — APPROVED** [review]
Looks great. This is the way astro images are MEANT to be viewed!

**Comment by stefanv** [comment]
@kmshin1397 I left one question, but if you think it's fine as-is I'm happy to merge.

File: static/js/components/RecentSources.jsx (hunk 1 above)
Comment (kmshin1397) [reply static/js/components/RecentSources.jsx]: @stefanv Good point. There were some other style rules already doing the same thing in ThumbnailList.jsx so I was just mindlessly following haha. Now fixed

**Review by stefanv — APPROVED** [review]

**Comment by stefanv** [comment]
On a roll, @kmshin1397 !

**Pull request merged**